apps/hubble-stats/components/charts/types.ts

```diff
@@ -27,3 +27,18 @@ export interface VolumeChartProps
     withdrawal: number;
   }>;
 }
+
+export interface ChartTooltipContentProps {
+  date: Date;
+  info: Array<{
+    color: string;
```

Comment:
Spacing is off

Reply:
The CI for Clean/Lint runs fine @drewstone!